x-pack/plugins/runtime_fields/public/components/runtime_field_form/runtime_field_form.tsx

```diff
@@ -119,7 +123,7 @@ const RuntimeFieldFormComp = ({ defaultValue, onChange, docsBaseUri }: Props) =>
               fullWidth
             >
               <CodeEditor
-                languageId="painless"
+                languageId={PainlessLang.ID}
                 // 99% width allows the editor to resize horizontally. 100% prevents it from resizing.
```

Comment:
@sebelga can you explain how to reproduce this?
I've tried this at 100% and 99% but am seeing what seems to be identical behavior (likely missing something). Thanks!

Reply:
I think this might be inherited from https://github.com/elastic/kibana/pull/57538/files#diff-74f90896d62130f9108f9f6a68490f6cd9042ab39eaf4911145f5eb3b8de986bR18. It occurs in Painless Lab because the different panes (so tempted to riff on a pun here) resize as you change window width. One of those panes contains a `<CodeEditor>`.

Reply:
Actually, it was taken from the mapings editor "runtime" field type from https://github.com/elastic/kibana/pull/79940/files#diff-86ba24cfc405b9dded48e7e1eb210143eef125c734cbdc130f5bc3ddf95d2d6aR30

I will remove it here and in the mappings editor if there are not needed in those contexts. 👍